execution/exchange/coalesce.go

```diff
@@ -50,6 +50,9 @@ type coalesce struct {
 }
 
 func NewCoalesce(pool *model.VectorPool, opts *query.Options, batchSize int64, operators ...model.VectorOperator) model.VectorOperator {
+	if len(operators) == 1 {
```

Comment:
residue from debugging

Reply:
Yeah good catch. But now that I think about it, would it be a good optimization to have?

Reply:
oh yeah generally yes, but we can do that with the late coalesce PR i think